driver-core/src/main/com/mongodb/MongoInterruptedException.java

```diff
@@ -31,7 +33,7 @@
      * @param message the message
      * @param e the cause
      */
-    public MongoInterruptedException(final String message, final Exception e) {
+    public MongoInterruptedException(@Nullable final String message, final Exception e) {
```

Comment:
Not having `@Nullable` here resulted in such useless message as ["Interrupted"](https://github.com/mongodb/mongo-java-driver/blob/a36c642537721785c17636d625188f546f8dc6bf/driver-core/src/main/com/mongodb/connection/netty/NettyStream.java#L471) or ["Interrupted acquiring a permit to retrieve an item from the pool"](https://github.com/mongodb/mongo-java-driver/blob/a36c642537721785c17636d625188f546f8dc6bf/driver-core/src/main/com/mongodb/internal/connection/ConcurrentPool.java#L203) (in the `acquirePermit` method).